src/redux/data.js

```diff
@@ -44,6 +44,8 @@ import {
 } from '../references';
 import { ethereumUtils, isLowerCaseMatch } from '../utils';
 import { addCashUpdatePurchases } from './addCash';
+/* eslint-disable-next-line import/no-cycle */
```

Comment:
Is there a way we can prevent having circular dependencies?   `src/redux/data` => `src/redux/uniqueTokens` => `src/redux/data`?

Reply:
Not sure, I need to think about this. Data needs uniqueTokens in order to refetch from OpenSea on new transactions. UniqueTokens needs Data in order to update assets that have unique tokens filtered out (bc zerion currently doesn't distinguish between a normal token and an erc721 yet).